elasticai_api/common/data_shard_service.py

```diff
@@ -51,6 +51,7 @@ def __init__(
         shuffle=False,
         shuffle_shards=False,
         task_type=elasticai_api_pb2.TRAINING,
+        num_minibatches_per_shard=0,
```

Comment:
Is it better to set a positive integer for the default value?

Reply:
The default value 0 means that we will use the value set by master.